src/mono/mono/mini/interp/interp.c

```diff
@@ -3530,13 +3574,65 @@ static long total_executed_opcodes;
 
 #define LOCAL_VAR(offset,type) (*(type*)(locals + (offset)))
 
+/*
+ * Custom C implementations of the min/max operations for float and double.
```

Comment:
I wonder what perf change this makes on non wasm platforms. 
Also should this be separated into a PR ?

Reply:
Good question. In my testing it didn't appear to do much for the test suites I ran, but for both raytracers it makes a big difference.

I'm open to splitting it out into a separate PR, it's just here as a dependency because two of my performance tests don't really work well without it.